make/stratos/_env

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+git_root=$(git rev-parse --show-toplevel)
+. "${git_root}/.envrc"
+
+: "${STRATOS_URL:=https://github.com/cloudfoundry-incubator/stratos/releases/download/2.1.1/console-helm-chart-2.1.1-74090ba.tgz}"
```

Comment:
Version numbers should be in `bin/common/versions.sh` (or whatever the correct path to that file is)